Conversions/DateToDay.js

```diff
@@ -4,32 +4,13 @@
     The DateToDay method takes a date in string format and
     returns the name of a day. The approach behind this method
     is very simple, we first take a string date and check
-    whether their date is valid or not, if the date is valid
-    then we do this But apply the algorithm shown below. The
-    algorithm shown below gives us the number of the day and
-    finally converts it to the name of the day.
-
-    Algorithm & Explanation : https://cs.uwaterloo.ca/~alopez-o/math-faq/node73.html
+    whether their date is valid or not, if the date is not valid
+    then we return a type error, else we apply Zeller's congruence.
+    Algorithm & Explanation : https://www.geeksforgeeks.org/zellers-congruence-find-day-date/
 */
 
-// March is taken as the first month of the year.
-const calcMonthList = {
-  1: 11,
-  2: 12,
-  3: 1,
-  4: 2,
-  5: 3,
-  6: 4,
-  7: 5,
-  8: 6,
-  9: 7,
-  10: 8,
-  11: 9,
-  12: 10
-}
-
 // show the week day in a number : Sunday - Saturday => 0 - 6
-const daysNameList = { // weeks-day
+const daysNameDict = { // weeks-day
```

Comment:
Make this an array please: Keys are `0` to `6`.

Reply:
Changed it to an array.